portal/index.php

```diff
@@ -357,14 +357,14 @@ function enableVerifyBtn(){
                     </div>
                 </div>
                 <div class="form-row my-3">
-                    <label class="col-md-2 col-form-label" for="pass"><?php echo !$_SESSION['onetime'] ? xlt('Current Password') : ''; ?></label>
+                    <label class="col-md-2 col-form-label" for="pass"><?php echo empty($_SESSION['onetime']) ? xlt('Current Password') : ''; ?></label>
                     <div class="col-md">
-                        <input class="form-control" name="pass" id="pass" <?php echo $_SESSION['onetime'] ? 'type="hidden" ' : 'type="password" '; ?> autocomplete="none" value="<?php echo attr($_SESSION['onetime']);
-                        $_SESSION['password_update'] = $_SESSION['onetime'] ? 2 : 1;
+                        <input class="form-control" name="pass" id="pass" <?php echo $_SESSION['onetime'] ?? null ? 'type="hidden" ' : 'type="password" '; ?> autocomplete="none" value="<?php echo attr($_SESSION['onetime'] ?? '');
```

Comment:
this line threw me off a bit. would put parenthesis around the `($_SESSION['onetime'] ?? null)`, then more clear

Reply:
sorry for the confusion, i meant the line above (but also nicer on the line below :) )